src/py-opentimelineio/opentimelineio/schema/serializable_collection.py

```diff
@@ -31,7 +31,8 @@ def __repr__(self):
 
 
 @add_method(_otio.SerializableCollection)
-def each_child(self, search_range=None, descended_from_type=_otio.Composable):
+def each_child(self, search_range=None, descended_from_type=_otio.Composable,
```

Comment:
We marked these as deprecated in 0.14.  We should add an issue to remove these in 0.16.

Reply:
Thanks, good catch on the asserts! It's a bit embarrassing creating a bug and then copying and pasting it half a dozen times. :)

Reply:
New issue to remove the deprecated functions: https://github.com/AcademySoftwareFoundation/OpenTimelineIO/issues/1409